python/code/test/test_manifold_discriminant.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+from classificationmethods import ManifoldDiscrimantAnalysisParafacTucker as pt
+import numpy as np
+from numpy import random as random
+import scipy.stats as stats
+
+"""
+    Unit tests for python manifold discriminant analysis. 
+"""
+
+
+def generate_data(classes, length_of_data_set, shape=None):
+    end_padding = length_of_data_set % classes
+    number_of_copies = length_of_data_set // classes
+    output_classes = number_of_copies * list(range(classes)) + list(range(end_padding))
```

Comment:
A comment here about the effect of multiplying a list by an integer might be helpful.